modules/executive/components/VoteModal/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
```

Comment:
Moved from the previous voteModal file, most of the logic is the same, i removed one useMemo and just rendered normally